### ISSUE TITLE
Error "Not implemented on Web Platform" on iOS

### ISSUE THREAD

**Issue by timrosskamp** [capacitor-community/sqlite#87]
**Describe the bug**

I get the Error "{"result":false,"message":"Not implemented on Web Platform"}" when trying to use this plugin on a virtual device started by XCode. This shouldn't happen since I'm on iOS and not the Web Platform. I also tried this in Andoid Studio and I get a similar Error.

I'm trying to connect to an existing database which has been copied to the device. The file is located in `DOCUMENTS/databases/mydatabaseSQLite.db`.

The line `add(CapacitorSQLite.class);` has been added to my `MainActivity.java`.

I'm not using `vue-sqlite-hook`, because I'm working with sqlite outside of the vue framework. Also I'm using `2.9.x` since my project is on Capacitor 2.

**To Reproduce**

I'm using this code trying to get the plugin to work:

```js
const sqlite = new SQLiteConnection(CapacitorSQLite)

console.log(await sqlite.isDatabase("mydatabase"))
console.log(await sqlite.createConnection('mydatabase', false, "no-encryption", 1))
```

Logs in XCode console:

```
⚡️  [log] - in Web isDatabase {"database":"mydatabase"}
⚡️  [log] - {"result":false,"message":"Not implemented on Web Platform"}
⚡️  [log] - createConnection {"database":"mydatabase","encrypted":false,"mode":"no-encryption","version":1}
⚡️  [log] - null
```

Logs in Android Studio console:

```
I/Capacitor/Console: File: http://localhost/js/chunk-vendors.30ee9b04.js - Line 1 - Msg: in Web isDatabase [object Object]
I/Capacitor/Console: File: http://localhost/js/app.488bed05.js - Line 1 - Msg: {"result":false,"message":"Not implemented on Web Platform"}
I/Capacitor/Console: File: http://localhost/js/chunk-vendors.30ee9b04.js - Line 1 - Msg: createConnection [object Object]
    File: http://localhost/js/app.488bed05.js - Line 1 - Msg: null
```

**Comment by jepiqueau** [comment]
@timrosskamp Did you run `npx cap sync` before building your app?

**Comment by timrosskamp** [comment]
Yes I did. I just tried it again and rerun `ionic cap sync` and `ionic cap build ios`. Unfortunately the error persists.

**Comment by timrosskamp** [comment]
When I run: `console.log(await sqlite.echo('ping'))`
I get this in the logs:
```
⚡️  [log] - ECHO in Web plugin {"value":"ping"}
⚡️  [log] - {"value":"ping"}
```

**Comment by timrosskamp** [comment]
I got it to work!

This is the code that seems to work:

```js
import { Plugins } from '@capacitor/core'
import { SQLiteConnection } from '@capacitor-community/sqlite'

const { CapacitorSQLite } = Plugins
const sqlite = new SQLiteConnection(CapacitorSQLite)

console.log(await sqlite.isDatabase('databases/mydatabase'))
console.log(await sqlite.createConnection('databases/mydatabase', false, 'no-encryption', 1))
```

I guess using `CapacitorSQLite` from `@capacitor-community/sqlite` always refers to the web plugin, while in `Plugins` from `@capacitor/core` the native plugin can be accessed.

Also I had to add the folder `databases/` to the database name.

This produces the following result in the logs:

```
⚡️  To Native ->  CapacitorSQLite isDatabase 50412144
⚡️  TO JS {"result":true}
⚡️  [log] - {"result":true}
⚡️  To Native ->  CapacitorSQLite createConnection 50412145
databaseName: databases/mydatabaseSQLite.db 
database path /Users/[...]/Documents/databases/mydatabaseSQLite.db
⚡️  TO JS {"result":true}
⚡️  [log] - >>> in SQLiteDBConnection dbName databases/mydatabase
⚡️  [log] - {"dbName":"databases/mydatabase","sqlite":{}}
```

**Comment by jepiqueau** [comment]
@timrosskamp Hi, if you look at `https://github.com/jepiqueau/angular-sqlite-app-starter/blob/refactor/src/app/services/sqlite.service.ts` you have the way it should be implemented for the 2.9.x version of the plugin.
for the 3.0.0 you should look at `https://github.com/jepiqueau/angular-sqlite-app-starter/blob/master/src/app/services/sqlite.service.ts` 
and do the same for Vue and it should work 
You could also have a look at `https://github.com/jepiqueau/vue-sqlite-hook/blob/2.9.x/src/useSQLite.ts`
Anyhow you found your way. Thanks for using the plugin

**Comment by timrosskamp** [comment]
Oh, now I see what happend. I did accually look at `vue-sqlite-hook` to copy the code. My mistake was I didn't switch to the latest branch, so I copied outdated code.
Thanks for the development on this plugin!
